test/src/test/java/org/apache/accumulo/harness/SharedMiniClusterIT.java

```diff
@@ -48,6 +50,7 @@
  * a method annotated with the {@link org.junit.BeforeClass} JUnit annotation and {@link #stopMiniCluster()} in a method annotated with the
  * {@link org.junit.AfterClass} JUnit annotation.
  */
+@Category(MiniClusterOnlyTest.class)
```

Comment:
It surprises me that these annotations would be inherited. I know annotations aren't inherited in Java, but maybe JUnit handles that itself. I assume you've tested this.